cosmosclient/cosmosclient.go

```diff
@@ -59,21 +70,21 @@ const (
 	orderAsc = "asc"
 )
 
-type broadcastMode string
-
 // Client is a client to access your chain by querying and broadcasting transactions.
 type Client struct {
 	// RPC is Tendermint RPC.
-	RPC *HTTP
+	RPC      rpcclient.Client
```

Comment:
instead of using custom `rpc.HTTP`, i'm exposing the `WSEvents` as part of the `Client`